Command/StatusCommand.php

```diff
@@ -70,7 +63,7 @@ public function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
-        $table->render();
+        $table->render($output);
```

Comment:
`TableHelper` is deprecated and will be removed in 3.0 - we should instantiate it only if `Table` does not exist.